tools/decompress_baserom.py

```diff
@@ -233,19 +234,21 @@ def main():
     str_hash = get_str_hash(file_content)
     if str_hash != correct_str_hash:
         print(
-            f"Error: Expected a hash of {correct_str_hash} but got {str_hash}. The baserom has probably been tampered, find a new one"
+            f"Error: Expected a hash of {correct_str_hash} but got {str_hash}. The baserom has probably been tampered, find a new one",
+            file=sys.stderr,
         )
 
         if version == "gc-eu-mq-dbg":
             if str_hash == "32fe2770c0f9b1a9cd2a4d449348c1cb":
                 print(
-                    "The provided baserom is a rom which has been edited with ZeldaEdit and is not suitable for use with decomp. Find a new one."
+                    "The provided baserom is a rom which has been edited with ZeldaEdit and is not suitable for use with decomp. Find a new one.",
+                    file=sys.stderr,
                 )
 
         exit(1)
 
     # Write out our new ROM
-    print(f"Writing new ROM {uncompressed_path}.")
+    print(f"Writing new ROM {uncompressed_path}...")
```

Comment:
Personally I would prefer all human-readable output go to stderr instead of splitting it up into both stdout and stderr. But I can't really imagine what else we'd use stdout for so I don't really care

Reply:
I'd much prefer errors going to stderr, while natural program writing go to stdout. This way even if I redirect stdout to some file, I still get to see the errors (if there are any) front and center.

Reply:
Well if this tool did write something useful to stdout which I then redirect to a file, I would rather have "Writing new ROM" appear in my terminal instead of in my output file. Or if I redirect stderr to get a log file, I would rather have these log messages mixed in with the error messages there. But, it's really not important here

Reply:
Agreed it's not important here but I also think all report-thing messages should go to a single stream (either stdout or stderr, unless the program's purpose is writing to stdout in which case stderr)
If you want outout verbosity control it should be a command line argument